.github/workflows/build.yml

```diff
@@ -13,10 +14,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.17
+    - name: Set up Go 1.19
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17
+        go-version: 1.19
```

Comment:
Could we use version 1.22? It's the current version in the infrastructure repo.

Reply:
Good catch, have updated to 1.22